client/src/components/v2/CameraRecordingStatus.js

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from 'react';
+import { useChannel, emit } from 'api/v2/socket';
+import formatBytes from 'utils/formatBytes';
+// import { Card } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+/**
+ * @typedef {object} CameraRecordingStatusProps
+ * @property {'primary'|'secondary'} device Camera screen
+ */
+
+/**
+ *  Status of a camera
+ *
+ * @param {CameraRecordingStatusProps} props Props
+ * @returns {React.Component<CameraRecordingStatusProps>} Component
+ */
+export default function CameraRecordingStatus({ device }) {
+  /**
+   * Format a camera recording status payload into JSX
+   *
+   * @param {string} payload Payload structure is defined in https://www.notion.so/V3-MQTT-Topics-66e6715d0e1941ffaa82020e5868fbae. See /v3/camera/recording/status/<primary/>secondary>
```

Comment:
I think we should remove `notion` links because this repo is public LOL. Sorry about this.

Reply:
You can replace it with " 'V3 MQTT Topics' page on Notion". People will know to use the search function to find the page.

Reply:
Done
6f3143a